python/test/microbenchmark/launch_overhead.py

```diff
@@ -0,0 +1,98 @@
+"""
+Original code by @bertmaher; profiling added by @apgoucher
+"""
+
+import cProfile
+import pstats
+import time
+
+import numpy as np
+import torch
+
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def nop_args(
+    t1,
+    t2,
+    t3,
+    t4,
+    t5,
+    i1,
+    i2,
+    i3,
+    i4,
+    i5,
+    i6,
+    i7,
+    i8,
+    i9,
+    c1: tl.constexpr,
+    c2: tl.constexpr,
+    c3: tl.constexpr,
+    c4: tl.constexpr,
+    c5: tl.constexpr,
+):
+    pass
+
+
+def do_bench_walltime(fn):
+    print("Compiling...")
+    fn()
+    torch.cuda.synchronize()
+
+    start_time = time.perf_counter()
+    for _ in range(1000):
+        fn()
+    torch.cuda.synchronize()
+    end_time = time.perf_counter()
+    estimate_ms = end_time - start_time
+
+    n_repeat = max(1, int(100 / estimate_ms))
```

Comment:
I've been using this modification locally, 
```suggestion
    n_repeat = 10000
```

It makes it so the profile time directly translates into the per call time, so time = 0.001 ms means 0.1 us per call.